test_requirements.txt

```diff
@@ -1,15 +1,11 @@
--e .
-black~=22.3
-flake8
-isort
-mypy
+.
+./pulp-glue-ostree
+
+# Test requirements
 pygments
 pytest
+pytest-subtests
 python-gnupg
-shellcheck-py
 
-types-pygments
-types-PyYAML
-types-requests
-types-setuptools
-types-toml
```

Comment:
How comes they are not needed anymore?

Reply:
It was removed from pulp-cli-maven. I thought it is redundant here as well.

https://github.com/pulp/pulp-cli-maven/commit/c1408eaf0989c3555e3b3ac4c61a4c1e7dce17d7#diff-7135b2521a3dac9ffd5e0e43edd7d813eb2cb4fd3ee0a4c3bea7ec75177ab932

Reply:
Hmm maybe... As long as mypi does not complain...